src/main/resources/core/properties/archived_dataset_file_dao.xml

```diff
@@ -28,7 +28,7 @@
     <query>
         <name>findByJobUuidWithFileReference</name>
         <sql>SELECT * FROM ARCHIVED_DATASET_FILE
-            WHERE JOB_UUID=? AND FILE_REFERENCE IS NOT NULL
+            WHERE JOB_UUID=? AND FILE_REFERENCE != ''
```

Comment:
why is the file reference blank and not null? You should set the file reference to null instead of changing this.

Reply:
They are set to "". Same as how manual extracts set them when they are in queue.